webpack.config.ts

```diff
@@ -0,0 +1,51 @@
+const path = require('path');
+const webpack = require('webpack');
+
+module.exports = {
+  entry: './src/cli.ts',
+  module: {
+    rules: [
+      {
+        test: /\.tsx?$/,
+        use: 'ts-loader',
+        exclude: /node_modules/,
+      },
+      {
+        test: /\.node$/,
+        loader: 'node-loader',
+      },
+      {
+        test: path.resolve(__dirname, 'node_modules/simple-websocket/server.js'),
+        use: 'null-loader',
```

Comment:
add a comment here explaining the bundle is not going to be used for live-preview?

Reply:
:+1

Reply:
done